src/lists/list.jsx

```diff
@@ -98,29 +101,15 @@ const List = React.createClass({
         paddingBottom: 8,
         paddingTop: subheader ? 0 : 8,
```

Comment:
This is what it should looks like:
![capture d ecran 2016-01-28 a 12 13 15](https://cloud.githubusercontent.com/assets/3165635/12642833/8cc8cab6-c5b8-11e5-96eb-780d3c33aac6.png)